test/datatypes.test.js

```diff
@@ -179,6 +180,34 @@ describe('MySQL specific datatypes', function() {
       });
     });
   });
+  it('should create a model instance with geopoint type', function(done) {
+    var city1 = {
+      name: 'North York',
+      loc: {
+        lat: 43.761539,
+        lng: -79.411079,
+      },
+    };
+    var xcor, ycor;
+    City.create(city1, function(err, res) {
+      assert.ok(!err);
```

Comment:
FWIW, we use the following pattern in other LoopBack projects - it produces more helpful failure messages:

```diff
- assert.ok(!err);
+ if (err) return done(err);
```